test/utils-tests.jl

```diff
@@ -441,6 +441,22 @@ let
     @test dec2 ≈ [-56.87186126487889]
 end
 
+# Test precess_cd
+@testset "precess_cd" begin
+    @test precess_cd([[30 60]; [60 90]], 1950, 2000, [13, 8], [43, 23]) ≈
```

Comment:
Check out http://gnudatalanguage.sourceforge.net/ then, most AstroLib procedures work in GDL ;-)